src/_common/community/community.model.ts

```diff
@@ -198,6 +198,20 @@ export class Community extends Collaboratable(Model) {
 			'community'
 		);
 	}
+
+	async saveGameSort(gameIds: number[]) {
+		const response = await Api.sendRequest(
+			`/web/dash/communities/games/save-sort/${this.id}`,
+			gameIds,
+			{
+				noErrorRedirect: true,
+			}
+		);
+		if (response.success) {
+			this.assign(response.community);
+		}
+		return response;
+	}
```

Comment:
I don't like blindly accepting gameIds without being able to validate them in the context they are used.
If we do the change I suggested for the games field, we could just work off of the order of games in `this.games`.